airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -247,11 +244,11 @@ def execute(self, context: Dict):
         resource = hook.get_query(query_id=self.report_id)
         # Check if report is ready
         if resource["metadata"]["running"]:
-            raise AirflowException('Report {} is still running'.format(self.report_id))
+            raise AirflowException("Report {} is still running".format(self.report_id))
```

Comment:
Would you mind using f-string?

Reply:
Yes, sure. 
I wanted to use f-string but I remember that some static checks don't like f-strings. It was Pylint?

Reply:
Done.